### PR TITLE
Support xBounds and yBounds options in viz.line

### PR DESCRIPTION
**Pull request by dorarad**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1514,10 +1529,10 @@ function lineWrapper() {
 // and smart hiding if there are too many rows
 function table(obj, options) {
   //wpEditor is not present if not run in the browser
-  if (typeof(wpEditor) === 'undefined') {
-    console.log("viz.print: no wpEditor, not drawing");
-    return;
-  }
+  // if (typeof(wpEditor) === 'undefined') {
```
Comment (longouyang) [inline src/index.js]: I prefer that PRs do one thing. Can you move the table changes to another PR?

**Comment by longouyang** [comment]
Could you also document this in README.md?

File: src/index.js (hunk 1 above)
Comment (dorarad) [reply src/index.js]: Done.

**Comment by dorarad** [comment]
Documented.

File: src/index.js [hunk 2]
```diff
@@ -1528,11 +1528,11 @@ function lineWrapper() {
 // TODO: support a data frame structure as input
 // and smart hiding if there are too many rows
 function table(obj, options) {
-  //wpEditor is not present if not run in the browser
-  // if (typeof(wpEditor) === 'undefined') {
-  //   console.log("viz.print: no wpEditor, not drawing");
-  //   return;
-  // }
+  wpEditor is not present if not run in the browser
```
Comment (longouyang) [inline src/index.js]: this line should be commented out

Comment (dorarad) [reply src/index.js]: Sure. Done.
